### ISSUE TITLE
[Bug]: Connections remembered between projects

### ISSUE THREAD

**Issue by thomassandsor** [Power-Maverick/DataverseDevTools-VSCode#44]
### What happened?

When opening a different project, the connection that I setup from the last one is showing. I thought you mentioned something about the connections being related to pr "project"?

### What version of the tool are you using?

Latest beta

**Comment by Power-Maverick** [comment]
Few follow-up questions:
1. When opening in VSCode do you open two different folders?
2. Are they open at the same time?
3. What happens when you change the connection on one folder; does it affect the other folder?

This will help me replicate the issue on my end in order to resolve it.

**Comment by thomassandsor** [comment]
I added when one was open.. Then i close everything but "Github Desktop".

THen i change the project in desktop, and open it in VSC. The connection loads. 

I could potentially read the connection one more time with 100% certainty that all other projects are closed, but don't think that is the issue

**Comment by Power-Maverick** [comment]
I tried to replicate it on my machine and was not able to reproduce it. Do you have any steps to reproduce the issue?

https://user-images.githubusercontent.com/36135520/148703564-c82101ea-e4e1-4f9d-b1e8-383491510b0b.mp4

**Comment by thomassandsor** [comment]
So I might be misunderstanding the function. 
![image](https://user-images.githubusercontent.com/33664322/148704093-1bc952ba-ea1f-4b1f-a6d0-9191bfe12316.png)

I was looking at the connections page. This is where I see it appearing in all projects where it "doesn't" belong. 

I did see the same behavior as you when I loaded the projects. "blue line on the bottom showing connection status correctly. 

So the question is, will the page where I link the javascripts etc be a long list of connections if I have many of them for different customers?
